README.md

```diff
@@ -61,6 +61,7 @@ f141        f141_ntarraydouble.fbs            [OBSOLETE] A simple array of doubl
 f142        f142_logdata.fbs                  For log data, for example forwarded EPICS PV update
 f143        f143_structure.fbs                [OBSOLETE] Arbitrary nested data
 ev42        ev42_events.fbs                   Multi-institution neutron event data
+ev43        ev43_events.fbs                   Multi-institution neutron event data to support fast pulse rates
```

Comment:
I'd make that
`Multi-institution neutron event data from multiple pulses`
We could even specify that ev42 only contains a single pulse.

Reply:
That makes sense. I'll drop the facility data and change the READ.me description.